Source/ACE/Entity/Container.cs

```diff
@@ -14,6 +14,11 @@ public class Container : WorldObject
 
         private readonly object inventoryMutex = new object();
 
+        // <summary>
+        /// Time when this object will despawn, -1 is never.
+        /// </summary>
+        public double DespawnTime { get; set; } = -1;
```

Comment:
I'm inclined to think this should be moved to WorldObject.  This certainly applies to corpses, but it also applies to random items on the ground.